src/Langs/Langs.php

```diff
@@ -44,6 +45,7 @@ final class Langs
         'pt' => 'com',  // Portuguese from Portugal: "com"
         'ro' => 'cu',   // Romanian from Romania: "cu"
         'vi' => 'và',   // Vietnamese from Vietnam: "và"
+        'tr' => 'tr',   // Turkish from Turkey: "tr"
```

Comment:
I'm not sure if the connector should be 've'; this is the value used to interconnect two numbers, for example (in spanish), 'diez con doce', where the connector is 'con'.

Reply:
@rmunate I noticed the mistake and corrected it. Thank you.